explorer/app/views/Index/index.tsx

```diff
@@ -31,26 +31,10 @@ function renderSummary(
   if (!summaryConfig || !summaryResponse) {
     return;
   }
-  /* Grab the "Summaries" component. */
```

Comment:
Hey @frano-m ! I think you should take a look at this to be sure that I'm not messing too much with something you did.
But I see three issues here:
- I think we should avoid these kind of meta-programming checks (`(component) => component.component.name === "Summaries"`), cause this makes the impact of changes too difficult to predict, and for this specific case we can't guarantee the function's name, since the file will be minified. 
- Don't think we should use `transformer` function outside `ComponentCreator`s. Imo it seems like spreading configuration code to "normal" components and possibly making things more complex.
- Not rendering the summary while we don't have a response from the summary request will make the component suddenly appear, which seems a bad UX, instead of presenting it with default values until we get the response.

Let me know what you think. I might be missing some problems that you have run into.